minecraft_servers/mush/manifest.json

```diff
@@ -4,7 +4,8 @@
   "direct_ip": "jogar.mush.com.br",
   "server_wildcards": [
     "%.mush.com.br",
-    "%.mushmc.com"
+    "%.mushmc.com",
+    "%.muu.sh"
```

Comment:
Not a real wildcard
https://i.imgur.com/SL6r8Yc.png

Reply:
Wildcard should be working now.